### ISSUE TITLE
hello,nice work for face recognition!have you tried all the loss function that you list?which is best so far?

### ISSUE THREAD

**Issue by FelixZhang7** [cavalleria/cavaface.pytorch#7]

**Comment by cavalleria** [comment]
thx, I have not compare the performance of all loss functions.
